infra/resources/_modules/messages_eventhub/messages_eventhub.tf

```
@@ -0,0 +1,38 @@
+module "messages-evh" {
+  source = "github.com/pagopa/dx//infra/modules/azure_event_hub?ref=3f205e62474782678a563f4cff92e479a34feecd"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location
+    domain          = var.domain
+    app_name        = "etl"
+    instance_number = var.instance_number
+  }
+
+  eventhubs = [{
+    name                   = "messages-evh"
```

```suggestion
    name                   = "io-p-com-etl-messages-evh-01"
```

(I think that `io-p-com-etl` can be obtained from variables)

---

the eventhub name is composed by the eventhub dx module. As you can see from the terraform plan the entire name is **io-p-itn-com-etl-messages-evh-01**.